.docker-compose.datagov-theme.yaml

```diff
@@ -28,7 +28,7 @@ services:
           install_standard_ckan_extension_github -r GSA/ckanext-geodatagov &&\
           install_standard_ckan_extension_github -r akariv/ckanext-datajson -b datagov &&\
           install_standard_ckan_extension_github -r akariv/USMetadata -e ckanext-usmetadata &&\
-          install_standard_ckan_extension_github -r GSA/ckanext-datagovtheme
+          install_standard_ckan_extension_github -r thejuliekramer/ckanext-datagovtheme
```

Comment:
This should be the canonical GSA no? Maybe just tag this PR as dependent on https://github.com/GSA/datagov-ckan-multi/issues/204?